projects/vdk-plugins/vdk-server/src/vdk/plugin/server/installer.py

```diff
@@ -709,6 +709,8 @@ def __helm_install_command(self, git_server_ip):
             "datajobTemplate.template.spec.successfulJobsHistoryLimit=5",
             "--set",
             "datajobTemplate.template.spec.failedJobsHistoryLimit=5",
+            "--set",
+            "deploymentBuilderImage.tag=1.2.3",
```

Comment:
why not 1.4.1?

Reply:
answered in desc. 

```
However importantly the new versions don't work with VDK server.
VDK server uses a git without ssl enabled and this causes issues for the new jobs builders.
SO for the moment vdk server will keep using the old version.
```

Reply:
You can link the ticket for resolving the server issue here